event-publisher/src/main/java/uk/gov/ons/ctp/common/event/model/SurveyUpdate.java

```diff
@@ -32,4 +33,10 @@ void setSampleDefinitionFromJson(JsonNode data) {
   void setMetadataFromJson(JsonNode data) {
     this.metadata = data.toString();
   }
+
+  private List<SurveyFulfilment> allowedPrintFulfilments;
```

Comment:
member variables would normally be placed above the methods

Reply:
Have changed order